GeoHealthCheck/plugins/probe/wcs.py

```diff
@@ -0,0 +1,130 @@
+from GeoHealthCheck.probe import Probe
+from owslib.wcs import WebCoverageService
+
+
+class WcsGetCoverage(Probe):
+    """
+    Get WCS coverage image using the OGC WCS GetCoverage v2.0.1 Operation.
+    """
+
+    NAME = 'WCS GetCoverage v2.0.1'
+    DESCRIPTION = """
+    Do WCS GetCoverage v2.0.1 request with user-specified parameters
+    for single Layer.
+    """
+    RESOURCE_TYPE = 'OGC:WCS'
+
+    REQUEST_METHOD = 'GET'
+    REQUEST_TEMPLATE = '?SERVICE=WCS&VERSION=2.0.1' + \
+                       '&REQUEST=GetCoverage&COVERAGEID={layer}' + \
+                       '&FORMAT={format}' + \
+                       '&SUBSET=x({subset[0]},{subset[2]})' + \
+                       '&SUBSET=y({subset[1]},{subset[3]})' + \
+                       '&SUBSETTINGCRS={subsetting_crs}' + \
+                       '&WIDTH={width}&HEIGHT={height}'
+
+    PARAM_DEFS = {
+        'layer': {
```

Comment:
Name should be `layers` not `layer` (similar to wms.py) , see also the test resources data.